### PR TITLE
fix: fixed chapter 12 and 13

### LINKED ISSUE #320: Content changes and Inline code block of JS Course -- Chapter 12 and 13.
Content changes and Inline code block of JS Course -- Chapter 12 and 13.

**Steps To Reproduce**
1. Go to https://test-app.nullcast.dev/curriculum
2. Click on the Hyperlink "School of Ducks" in the header.
3. Scroll down to the Curriculum.
4. Click on "Learn More" button of Javascript, where User gets redirected to the Javascript Course page.
5. Click on the "Start" button, where the users gets redirected to the Javascript chapters page.
6. Click on the Bottom Hamburger icon where the Nav bar containing the menu of all the chapters listed.
7. User while learrning through the chapters 1 and 2, could see some issues in contents and some technical terms which need to be given the Inline code block (single back tick), which are marked inside boxes.

**Expected behavior**
Content Changes and the Inline code block to be corrected in the JS chapters 12 and 13.

**Screenshots attached**

**Desktop (please complete the following information):** ThinkPad
OS: Linux
Browser: Google Chrome
Version: Version 95.0.4638.54 (Official Build) (64-bit)

**Additional context**
Tested in Testsite
13Arrow-Functions2
13Arrow-Functions1
12Type-conversion3
12Type-conversion2
12Type-conversion1

### PR DESCRIPTION
**Pull request by kunwar-11**
# Pull Request Template

## Description

fixed Content changes and Inline code block of JS Course for chapter 12 and 13

Fixes #320 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by misuvi** [comment]
Resolve conflicts

**Comment by misuvi** [comment]
@kunwar-11 Resolve the conflicts and tag me here

**Comment by kunwar-11** [comment]
> @kunwar-11 Resolve the conflicts and tag me here

resolved

File: courses/javascript/arrow-functions.md [hunk 1]
```diff
@@ -49,60 +49,56 @@ testCase: [
...
-const namesLength = names.map(function(name) { return name.length; });
+const names = ["Kratos", "Mario", "Nathan", "Lara"];
+
+const namesLength = names.map(function (name) {
+  return name.length;
+});
 console.log(namesLength);
 ```
+
 Using _arrow functions_ :
```
Comment (misuvi) [inline courses/javascript/arrow-functions.md:90]: inline code block here

File: courses/javascript/type-conversion.md [hunk 2]
```diff
@@ -3,138 +3,155 @@ title: "Type conversions in Javascript"
...
+      hint: 'a should have a value of "45" or ''45'' and b should have a value of "32" or ''32''',
+      isCorrect: false
     },
     {
-        id: 2,
-        case: ['const sum = a + b;'],
-        hint: "sum should store the result of performing an ADD operation on the numb",
-        isCorrect: false
+      id: 2,
+      case: ["const sum = a + b;"],
```
Comment (misuvi) [inline courses/javascript/type-conversion.md:24]: add a case without semicolon

File: courses/javascript/type-conversion.md [hunk 3]
```diff
@@ -3,138 +3,155 @@ title: "Type conversions in Javascript"
...
+  [
     {
-        id: 1,
-        case: ["const a = '45', b = '32';", "const a = '45', b = '32'","const a = '45'; const b = '32';", "const a = \"45\", b = \"32\";", "const a = \"45\", b = \"32\"","const a = \"45\"; const b = \"32\";"],
-        hint: "a should have a value of \"45\" or '45' and b should have a value of \"32\" or '32'",
-        isCorrect: false
+      id: 1,
+      case:
+        [
+          "const a = '45', b = '32';",
```
Comment (misuvi) [inline courses/javascript/type-conversion.md:12]: Add cases without semicolon too

File: courses/javascript/type-conversion.md [hunk 4]
```diff
@@ -3,138 +3,155 @@ title: "Type conversions in Javascript"
...
+      hint: "sum should store the result of performing an ADD operation on the numb",
+      isCorrect: false
     },
     {
-        id: 3,
-        case: ["const result = Number(sum);"],
-        hint: "Please re check the code and formatting.",
-        isCorrect: false
+      id: 3,
+      case: ["const result = Number(sum);"],
```
Comment (misuvi) [inline courses/javascript/type-conversion.md:30]: Add cases without semicolon too

File: courses/javascript/type-conversion.md [hunk 5]
```diff
@@ -3,138 +3,155 @@ title: "Type conversions in Javascript"
...
+Number("99 88"); // returns NaN
 ```
 
 ## Boolean Conversion
 
 The conversion rule:
 
-Values that are intuitively “empty”, like 0, an empty string, null, undefined, and NaN, become false.
-Other values become **true**.
+Values that are intuitively “empty”, like 0, `an empty string`, `null`, `undefined`, and `NaN`, become `false`.
```
Comment (misuvi) [inline courses/javascript/type-conversion.md:132]: change `an empty string` to an empty string `''`

**Review by misuvi — APPROVED** [review]

**Pull request merged**